api-server-nest/src/shared-types/db-delete-response.ts

```diff
@@ -0,0 +1,7 @@
+export default interface DbDeleteResponse {
+    numberOfAffectedEntities: number
+}
+
+export function toDbDeleteResponse(response: number): Promise<DbDeleteResponse> {
+    return Promise.resolve({ numberOfAffectedEntities: response })
+}
```

Comment:
I was wondering if we need to wrap in the Promise.resolve or if that's implicit when you return in a .then context.

I've been doing this and was thinking we needed to but then I did some tests on our project at work that seemed to indicate it was superfluous.

It could be dependent on webpack config. What have you seen in this regard @Mathew?